### DISCUSSION TITLE
Auto-reload issues with the application

### DISCUSSION THREAD

**Discussion by LuanMiranda77** [saltyshiomix/nextron#404] [General]
I used the command 'yarn create nextron-app my-app --example with-tailwindcss' to generate a project for experimenting and learning Next.js. However, I'm facing difficulties as the project isn't auto-reloading the page when I make changes to the app's pages. Even when I use the 'force reload' option in the Electron window, the application only updates the changes if I restart the server. Can anyone help me with this?"

**Comment by bm777** [comment]
Hi @LuanMiranda77, it looks like you have an issue with `TailwindCSS`.  

verify that your Next.js configuration (typically in next.config.js) and Tailwind CSS configuration (usually in tailwind.config.js) are correctly set up. Ensure that there are no syntax errors or misconfigurations. (Maybe you made some change somewhere by mistake).
